spark/components/buttons/vanilla/button.stories.js

```diff
@@ -1,48 +1,69 @@
-import { withKnobs, text } from '@storybook/addon-knobs';
 
 export default {
   title: 'Components|Buttons',
-  decorators: [withKnobs],
 };
 
 export const primary = () => (
   `
     <button class="sprk-c-Button" data-id="button-1">
-      ${text('button text', 'Button')}
+      Button
     </button>
   `
 );
 
 export const secondary = () => (
-  '<button class="sprk-c-Button sprk-c-Button--secondary" type="button" data-id="button-1">Button</button>'
+  `
+    <button class="sprk-c-Button sprk-c-Button--secondary" type="button" data-id="button-1">
```

Comment:
The data-ids on these buttons should be unique because in SB they are all displayed together on the "Docs" panel.

Reply:
done!